src/Analyzers/Core/Analyzers/EnforceOnBuildValues.cs

```diff
@@ -0,0 +1,113 @@
+        public const EnforceOnBuild UsePatternCombinators = /*IDE0078*/ EnforceOnBuild.Recommended;
+        public const EnforceOnBuild RemoveUnnecessaryByVal = /*IDE0081*/ EnforceOnBuild.Recommended;
+        public const EnforceOnBuild ConvertTypeOfToNameOf = /*IDE0082*/ EnforceOnBuild.Recommended;
+        public const EnforceOnBuild UseNotPattern = /*IDE0083*/ EnforceOnBuild.Recommended;
+        public const EnforceOnBuild UseIsNotExpression = /*IDE0084*/ EnforceOnBuild.Recommended;
+        public const EnforceOnBuild UseImplicitObjectCreation = /*IDE0090*/ EnforceOnBuild.Recommended;
+        public const EnforceOnBuild RemoveRedundantEquality = /*IDE0100*/ EnforceOnBuild.Recommended;
+        public const EnforceOnBuild RemoveUnnecessaryDiscardDesignation = /*IDE0110*/ EnforceOnBuild.Recommended;
+        public const EnforceOnBuild InvokeDelegateWithConditionalAccess = /*IDE1005*/ EnforceOnBuild.Recommended;
+        public const EnforceOnBuild NamingRule = /*IDE1006*/ EnforceOnBuild.Recommended;
+
+        /* EnforceOnBuild.WhenExplicitlyEnabled */
+        public const EnforceOnBuild RemoveUnnecessaryCast = /*IDE0004*/ EnforceOnBuild.WhenExplicitlyEnabled; // TODO: Move to 'Recommended' OR 'HighlyRecommended' bucket once performance problems are addressed: https://github.com/dotnet/roslyn/issues/43304
+        public const EnforceOnBuild PopulateSwitchStatement = /*IDE0010*/ EnforceOnBuild.WhenExplicitlyEnabled;
+        public const EnforceOnBuild UseInferredMemberName = /*IDE0037*/ EnforceOnBuild.WhenExplicitlyEnabled;
+        public const EnforceOnBuild UseIsNullCheck = /*IDE0041*/ EnforceOnBuild.WhenExplicitlyEnabled;
+        public const EnforceOnBuild AddRequiredParentheses = /*IDE0048*/ EnforceOnBuild.WhenExplicitlyEnabled;
+        public const EnforceOnBuild ExpressionValueIsUnused = /*IDE0058*/ EnforceOnBuild.WhenExplicitlyEnabled;
+        public const EnforceOnBuild MakeStructFieldsWritable = /*IDE0064*/ EnforceOnBuild.WhenExplicitlyEnabled;
+        public const EnforceOnBuild ConvertSwitchStatementToExpression = /*IDE0066*/ EnforceOnBuild.WhenExplicitlyEnabled;
+        public const EnforceOnBuild DisposeObjectsBeforeLosingScope = /*IDE0067*/ EnforceOnBuild.WhenExplicitlyEnabled;
+        public const EnforceOnBuild UseRecommendedDisposePattern = /*IDE0068*/ EnforceOnBuild.WhenExplicitlyEnabled;
+        public const EnforceOnBuild DisposableFieldsShouldBeDisposed = /*IDE0069*/ EnforceOnBuild.WhenExplicitlyEnabled;
+        public const EnforceOnBuild PopulateSwitchExpression = /*IDE0072*/ EnforceOnBuild.WhenExplicitlyEnabled;
+        public const EnforceOnBuild Regex = /*RE0001*/ EnforceOnBuild.WhenExplicitlyEnabled;
+
+        /* EnforceOnBuild.Never */
+        // TODO: Allow enforcing simplify names and related diagnostics on build once we validate their performance charactericstics.
+        public const EnforceOnBuild SimplifyNames = /*IDE0001*/ EnforceOnBuild.Never;
```

Comment:
💭 I'd almost prefer a comment like this:

```suggestion
        /// <seealso cref="IDEDiagnosticIds.SimplifyNamesDiagnosticId"/>
        public const EnforceOnBuild SimplifyNames = EnforceOnBuild.Never;
```

Reply:
Sure, will do.

Reply:
That seems to degrade the readability with nice grouping structure in this file, I am going to leave this as-is for now.

## Current
![image](https://user-images.githubusercontent.com/10605811/102412710-b0e84600-3fa8-11eb-986c-d794e769efe5.png)


## Suggested
![image](https://user-images.githubusercontent.com/10605811/102412822-da08d680-3fa8-11eb-8bce-22b76c0c5f8c.png)